
8000 8100 2900
[2023-09-08 11:14:24,258][main.py][line:44][INFO] loading pretrained checkpoint from ./ckpt/original__8636.pkl.
[2023-09-08 11:14:24,281][main.py][line:44][INFO] loading pretrained checkpoint from ./ckpt/ucf_trans_2022.pkl.
[2023-09-08 11:14:24,288][main.py][line:284][INFO] total params:1.2091M
[2023-09-08 11:14:24,289][main.py][line:287][INFO] Training Mode
[2023-09-08 11:14:29,137][main.py][line:151][INFO] Random initialize AUCAUC:0.8636 Anomaly AUC:0.69742
[2023-09-08 11:14:33,167][main.py][line:154][INFO] Random initialize AUCAUC:0.8695 Anomaly AUC:0.70773
Predict video 10 | num abnormal video:0:   0%|‚ñè                                                                                                                                                                   | 9/8100 [00:00<02:55, 46.06it/s]
Traceback (most recent call last):
  File "/home/yukaneko/dev/AbnormalDetection/PEL4VAD/main.py", line 323, in <module>
    main(cfg)
  File "/home/yukaneko/dev/AbnormalDetection/PEL4VAD/main.py", line 289, in main
    train(pel_model, ur_model, train_nloader, train_aloader, test_loader, gt, logger)
  File "/home/yukaneko/dev/AbnormalDetection/PEL4VAD/main.py", line 160, in train
    index_list, frame_num_dict = make_new_dataset(pel_model, ur_model, train_aloader, 0.9)
  File "/home/yukaneko/dev/AbnormalDetection/PEL4VAD/main.py", line 110, in make_new_dataset
    frame_num_dict[Idlist[0]] = index_l.cpu()
NameError: name 'Idlist' is not defined